akka-actor/src/main/scala/akka/event/LoggerMailbox.scala

```diff
@@ -43,6 +43,5 @@ private[akka] class LoggerMailbox(owner: ActorRef, system: ActorSystem)
         envelope = dequeue
       }
     }
-    super.cleanUp(owner, deadLetters)
```

Comment:
@patriknw no I haven't seen it, I didn't know exactly when cleanUp would be called but I was thinking that maybe a message would at been inserted between end of loop and call to super (a very small chance), what you are saying that chance is then zero and that call is completely harmless, though I would still remove it as it is misleading.

WDYT?